apiary.apib

```diff
@@ -4421,6 +4422,8 @@ Get a list of available payment terms.
             + (object)
                 + id: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
                 + Include PaymentTerm
+        + meta (object)
+            + `default`: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
```

Comment:
Let's make sure we align this property structure with all future ones we'd do, e.g. dealPipelines cc @gdegirmenci